server/src/proxyManagement.mjs

```diff
@@ -0,0 +1,191 @@
+/*
+* Copyright 2021 Comcast Cable Communications Management, LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+'use strict';
+
+import promisify from 'util';
+import child_process from 'child_process';
+const exec = promisify.promisify(child_process.exec);
+import { parse } from 'url';
+import WebSocket from 'ws';
+let websocketConnection = null 
+
+async function initialize(receiverWSClient) {
+    const url = await buildWSUrl()
+    if (url) {
+      console.log("Establishing websocket connection")
```

Comment:
Can this be a little more descriptive considering we now have several connections to deal with? "Downstream" or "Proxy" connections come to mind.

Reply:
Addressed